pretty_midi/pretty_midi.py

```diff
@@ -219,14 +217,11 @@ def _load_metadata(self, midi_data):
                         event.text, self.__tick_to_time[event.time]))
                     
             if lyrics:
-                tracks_with_lyrics.append(lyrics)
+                self.lyrics[track.name] = lyrics
```

Comment:
Are track names guaranteed to be unique? OTOH I don't think they are so I'm not sure we should key on them.

Reply:
I agree. I changed it to key on the index of the midi tracks, in consonance to how is done in other parts of the code.